lang/java/build.sh

```diff
@@ -38,7 +38,7 @@ for target in "$@"
 do
 
 function do_dist() {
-  mvn package -DskipTests -Dhadoop.version=1;
+  mvn package -DskipTests;
   rm -rf mapred/target/{classes,test-classes}/;
   rm -rf trevni/avro/target/{classes,test-classes}/;
   mvn -P dist package -DskipTests -Davro.version=$VERSION javadoc:aggregate
```

Comment:
since we're doing a package here on the last line, can we remove the first package call and the rm invocations?